target/linux/ath79/image/mikrotik.mk

```diff
@@ -4,7 +4,7 @@ define Device/mikrotik_routerboard-493g
   $(Device/mikrotik_nand)
   SOC := ar7161
   DEVICE_MODEL := RouterBOARD 493G
-  DEVICE_PACKAGES += kmod-usb-ohci kmod-usb2
+  DEVICE_PACKAGES += kmod-usb-ohci kmod-usb2 kmod-switch-ar8xxx kmod-swconfig
```

Comment:
You do not have to specify `kmod-swconfig`, it will be automatically added because `kmod-switch-ar8xxx` depends on it.